### PR TITLE
[FD] - Product tile

### PR DESCRIPTION
**Pull request by KonstantinChabanenko**
Product tile component

### CODE REVIEW

File: src/client/components/ProductTile/Prices.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useMemo } from 'react';
+
+const Prices = ({ product }) => {
+  const { priceMin, priceMax, variationProducts, currencySymbol } = product;
+  const memoizedListPrice = useMemo(() => {
```
Comment (KorenYa) [inline src/client/components/ProductTile/Prices.jsx]: Consider creating shared helper to get price range from `variationProducts`

File: src/client/components/ProductTile/ProductTile.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import Swatches from './Swatches';
+import currencies from '../../static/currencies';
+
+const ProductTile = () => {
+  const [product, setProduct] = useState();
+  const { title, imageSrc, imageAlt, variationProducts, variationAttributes } = product || {};
+  
+  useEffect(() => {
+    getProduct("25518058M", { expand: "variations,images,prices" }).then(res => {
+      console.log(res);
```
Comment (KorenYa) [inline src/client/components/ProductTile/ProductTile.jsx]: Avoid debugging in code ready for PR.

File: src/client/components/ProductTile/ProductTile.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <div className="product-tile">
+      <div className="image-container">
+        <Link to="/product-tile">
+          <img src={imageSrc} alt={imageAlt}/>
+        </Link>
+      </div>
+      <div className="product-tile__body">
+        <Swatches variationProducts={variationProducts} variationAttributes={variationAttributes} />
+        <div className="product-tile__pdp-link">
+    <Link to="/product-tile">{title}</Link>
```
Comment (KorenYa) [inline src/client/components/ProductTile/ProductTile.jsx]: As suggestion - you can use formatOnSave option of your IDE to auto-handle spaces.

File: src/client/components/ProductTile/ProductTile.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import Prices from "./Prices";
+import Swatches from './Swatches';
+import currencies from '../../static/currencies';
+
+const ProductTile = () => {
+  const [product, setProduct] = useState();
+  const { title, imageSrc, imageAlt, variationProducts, variationAttributes } = product || {};
+  
+  useEffect(() => {
+    getProduct("25518058M", { expand: "variations,images,prices" }).then(res => {
```
Comment (KorenYa) [inline src/client/components/ProductTile/ProductTile.jsx]: Were you thinking to use actions/middleware/redux as alternative to making API calls directly in components?

**Comment by KonstantinChabanenko** [comment]
I will create another PR with both "product tiles grid" and "product tile" components

**Pull request closed without merging**